packages/util/src/obj.ts

```diff
@@ -118,7 +118,7 @@ export const getAnyKey = function(obj) {
 };
 
 export const getValues = function(obj) {
-  var res = [];
+  var res: any[] = [];
```

Comment:
Can we replace this whole function with `Object.values()`?

Reply:
Probably. The only difference is that `Object.values()` doesn't get properties in the prototype chain? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values But in the one case is invoked, that doesn't seem to matter. I don't want to mess with functionality too much in a TS PR though, maybe next time.

Reply:
Getting properties from the prototype chain sounds like a great footgun TBH. :smile:

I followed the calls to this function, and the only reason it's not caused an error yet is because everywhere it's used, there's a check like `if (change.type === some_type)` before any other field is accessed, and luckily, this just results in a check like `undefined === some_type` for the values and functions from the prototype chain.

I understand that you don't want to change this in this PR, but this is very likely to cause an error, and we should definitely change it.

I can send a PR if you'd like.

Reply:
Sure, send a PR, sounds good.